packages/grid/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@undercoatcss/grid",
+  "version": "0.0.0",
+  "description": "Grid component for the Undercoat CSS Framework",
+  "keywords": [
```

Comment:
Not sure but think we can leave only "CSS" and "framework" and exclude the "CSS framework"

Reply:
I'll leave it as is for now.